charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Datadog changelog
 
+## 3.59.7
+
+* Add configuration option to specify clusterAgent.admissionController.containerRegistry, which defaults to registry
+* Add configuration option to specify clusterAgent.admissionController.autoInstrumentation.containerRegistry
```

Comment:
Do we need both `clusterAgent.admissionController.autoInstrumentation.containerRegistry` and `clusterAgent.admissionController.containerRegistry`?

Reply:
Thinking the same thing, will anyone actually use both? I can drop it.